app/assets/stylesheets/_resources.scss

```diff
@@ -1,4 +1,4 @@
-.resources_index {
+.resources {
```

Comment:
Either way is fine, as long as we're consistent and not breaking all the pretty styles Connie has put into place!